src/js/tabs/osd.js

```diff
@@ -434,6 +434,19 @@ OSD.drawCameraFramePreview = function() {
     return cameraFrame;
 };
 
+OSD.formatPidsPreview = function(axis) {
+    const pidDefaults = FC.getPidDefaults();
+    const p = pidDefaults[axis * 5].toString().padStart(3);
+    const i = pidDefaults[axis * 5 + 1].toString().padStart(3);
+    const d = pidDefaults[axis * 5 + 2].toString().padStart(3);
+    const f = pidDefaults[axis * 5 + 4].toString().padStart(3);
+    if (semver.lt(FC.CONFIG.apiVersion, "1.44.0")) {
```

Comment:
```suggestion
    if (semver.lt(FC.CONFIG.apiVersion, API_VERSION_1_44) {
```

Reply:
I'm not really sure how that's an improvement. There are 54 occurrences of a version check in just this file alone and only 6 of them use the suggested constant style. It seems to me that this adds unnecessary fragmentation. If a follow-on PR wants to address this and change to a consistent style throughout the code then that's fine with me, but out of scope for this PR.

Reply:
The constant was introduced with API_VERSION_1_43

Reply:
We are adding the constants to any new code. I can do later a PR to change all of the old occurrences, but if we are adding new code is better to add the constant at this PR.

Reply:
Ok, updated.

Reply:
Thanks!

Reply:
It's just avoidance of magic numbers - in general they are bad coding style.